test/drb/test_drb.rb

```diff
@@ -305,11 +305,18 @@ def test_01_large_ary
     ary = [2] * 10240
     assert_equal(10240, @there.size(ary))
     assert_equal(20480, @there.sum(ary))
+    assert_equal(2 ** 10240, @there.multiply(ary))
+    assert_equal(2, @there.avg(ary))
+    assert_equal(2, @there.median(ary))
   end
 
   def test_02_large_ary
     ary = ["Hello, World"] * 10240
     assert_equal(10240, @there.size(ary))
+    assert_equal(ary[0..ary.length].inject(:+), @there.sum(ary))
+    assert_raise_with_message(TypeError, "no implicit conversion of String into Integer") {@there.multiply(ary)}
+    assert_equal(0.0, @there.avg(ary))
```

Comment:
This is inconsistent with the behaviour of muliplying a string array which throws a TypeError.
Why would the average of a string array be 0?